subsys/bluetooth/mesh/net.c

```diff
@@ -98,8 +96,16 @@ struct bt_mesh_net bt_mesh = {
  */
 static bool ivi_was_recovered;
 
-NET_BUF_POOL_DEFINE(loopback_buf_pool, CONFIG_BT_MESH_LOOPBACK_BUFS,
-		    LOOPBACK_MAX_PDU_LEN, LOOPBACK_USER_DATA_SIZE, NULL);
+struct loopback_buf {
+	sys_snode_t node;
+	struct bt_mesh_subnet *sub;
+	uint8_t len;
+	uint8_t data[LOOPBACK_MAX_PDU_LEN];
+};
+
+K_MEM_SLAB_DEFINE(loopback_buf_pool,
+		  sizeof(struct loopback_buf),
+		  CONFIG_BT_MESH_LOOPBACK_BUFS, __alignof__(struct loopback_buf));
```

Comment:
Is there any reason to align buffers to `struct loopback_buf`? Will it be enough to align to machine word size?

Reply:
I think this method is more compatible.

Reply:
I tried both boundaries. Seems there is no difference in RAM size. I see it is used similarly in Bluetooth host but aligned onto hardcoded 4 bytes in drivers, fs and etc. No common usage way among components.
Then LGTM.